Logging/src/V2/ConfigServiceV2Client.php

```diff
@@ -21,10 +21,6 @@
  * https://github.com/google/googleapis/blob/master/google/logging/v2/logging_config.proto
  * and updates to that file get reflected here through a refresh process.
  *
- * EXPERIMENTAL: This client library class has not yet been declared GA (1.0). This means that
```

Comment:
@michaelbausor I just wanted to verify with you, do you think we should be removing this notice?

Reply:
Yes, I believe we are now ready to remove these notices.

Reply:
What do we want to do here?

Reply:
Does removing this mean we are saying the Logging GAPIC is considered GA along with the veneer?